src/megachatapi.h

```diff
@@ -2630,6 +2631,14 @@ class MegaChatApi
      */
     void disableVideoCall(MegaChatHandle chatid, bool videoCall, MegaChatRequestListener *listener = NULL);
 
+    /**
+     * @brief Search all audio and video devices at the system at that moment.
+     * The associated request type with this request is MegaChatRequest::TYPE_LOAD_AUDIO_VIDEO_DEVICES
+     *
```

Comment:
Maybe this request could return in the request's object the new list of devices, it could be useful for the caller.